keymaps/intellij-idea-keymap.cson

```diff
@@ -113,13 +140,54 @@
   # Recent files popup
   'cmd-e': 'fuzzy-finder:toggle-buffer-finder'
 
+  # Reveal active file
+  'cmd-ctrl-s': 'tree-view:reveal-active-file'
+
+  # Show bookmarks
+  'cmd-f3': 'bookmarks:view-all'
+
+  # Toggle bookmark
+  'f3': 'bookmarks:toggle-bookmark'
+
   ##################
   # Live Templates #
   ##################
 
   # Insert live templates
   'cmd-j': 'snippets:available'
 
+'.platform-darwin .tree-view':
+
+  ###########
+  # General #
+  ###########
+
+  'left': 'tree-view:collapse-directory'
+  'right': 'tree-view:expand-item'
+  'enter': 'tree-view:open-selected-entry'
+  'cmd-n': 'tree-view:add-file'
+
+  ##############
+  # Navigation #
+  ##############
+
+  # Open corresponding tool window
+  'cmd-1': 'tree-view:toggle'
+
+  ###########
+  # Editing #
+  ###########
+
+  'cmd-c': 'tree-view:copy'
```

Comment:
This shortcut already works out of the box in Atom. I would remove it here such that we just have the shortcuts which are different between IDEA and Atom.

This is also the case for:

* `cmd-x`
* `cmd-v`
* `backspace`

The `delete` I cannot test as I only have the laptop keyboard. ;)